index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav>Ahmed Reda</nav>
+      <nav><img src="./Enabled.png" alt="menu" /></nav>
+    </header>
+    <div class="container">
+      <section class="first-section">
+        <h1>Ahmed Reda</h1>
+        <h2>Software Developer</h2>
+      </section>
+      <section class="seconed-section">
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </section>
```

Comment:
- [ ] Please use only one `section` tag to enclose all the code in this area of your portfolio. Since all of these elements "belong" to the Headline of your page they should be enclosed semantically by the same tag.